client/components/plans/plan/index.jsx

```diff
@@ -83,6 +88,11 @@ module.exports = React.createClass( {
 				'is-plan-specific': feature.planSpecific
 			} );
 
+			if ( abtest( 'plansFeatureList' ) === 'andMore' && feature.testVariable ) {
```

Comment:
Is `testVariable` a remnant of something else? It is slightly confusing to me to show the 'more link' based on a property on a specific feature (and the property appears on two features). It seems to me that we can get rid of it, and change the condition to show the 'more link' if:

- the user is in the `andMore` test
- the given plan is the business plan